packages/govuk-frontend/src/govuk/components/header/_index.scss

```
@@ -81,6 +91,11 @@
     &:last-child {
       margin-right: 0;
     }
+
+    @include _govuk-rebrand {
+      margin-right: 12px; // 3 'crown dots'
```

```suggestion
      margin-right: 7px; // 1 'dot'
```

I agreed with the brand team that we would have 7px space between the wordmark and the 'product name' with any additional whitespace that would come from the first letter. More info is in the [cross team decision making tab of this Google document](https://docs.google.com/document/d/1ehId5JD5Fc-pthBDlHX1mn-IwDM5Azi0375DUOsBiBk/edit?tab=t.3ha72dpcnvlv)